collect_app/src/main/java/org/odk/collect/android/widgets/RatingWidget.java

```diff
@@ -61,8 +61,8 @@ protected View onCreateAnswerView(Context context, FormEntryPrompt prompt, int a
         }
 
         binding.ratingBar1.setOnRatingBarChangeListener((ratingBar, rating, fromUser) -> {
-            binding.ratingBar1.setRating(rating);
             binding.ratingBar2.setRating(0.0F);
+            binding.ratingBar1.setRating(rating);
```

Comment:
Could this change have been driven out with a test? Maybe there was one and I missed it!

Reply:
Sorry @seadowg for missing the tests. I added unit tests and confirmed that unit test`changingRating_updatesAnswer_forRatingBarInMultipleLines()` failed for the last commit, and passes on the latest one.